front/components/welcome/WelcomeThird.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <v-content>
+    <welcome-desktop
+      class="hidden-mobile"
```

Comment:
there is classes that can be used already e.g. hidden-xs-only hidden-md-and-up et etc etc from vuetify which could be used instead of writing your onw

Reply:
I was trying to get these to work but they did not work how I expected them to. I will try this again. Would you advise not using the breakpoints in the variables.scss file if possible in favor of Vuetify?

Reply:
My issues were due to d-flex overriding the hide classes. It has been fixed.

Reply:
>Would you advise not using the breakpoints in the variables.scss file if possible in favor of Vuetify?

I already created ticket for this 
https://github.com/ezl/dailyidea.com/issues/415